documentation/docs/fastly:cache/SimpleCache/purge.mdx

```diff
@@ -0,0 +1,89 @@
+---
+hide_title: false
+hide_table_of_contents: false
+pagination_next: null
+pagination_prev: null
+---
+
+# SimpleCache.purge
+
+purge the entry associated with the key `key` from the cache.
+
+## Syntax
+
+```js
+purge(key, options)
+```
+
+### Parameters
+
+- `key` _: string_
+  - The key to purge from within the cache.
+
+- `options` _: object_
```

Comment:
If scope is the main required configuration, would it be worth considering a convenience form like `purge(key, 'global')` or `purge(key, 'local')`?